consensus/istanbul/core/core.go

```diff
@@ -618,8 +618,15 @@ func (c *core) createRoundState() (RoundState, error) {
 		roundState, err = c.rsdb.GetRoundStateFor(lastStoredView)
 
 		if err != nil {
-			logger.Error("Failed to fetch lastStoredRoundState", "err", err)
-			return nil, err
+			logger.Warn("Failed to fetch lastStoredRoundState", "err", err)
+			if err == leveldb.ErrNotFound {
```

Comment:
Can we maybe extract the common functionality of re-creating the round state to some function?